### PR TITLE
[LMS-316] LMS Extension

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: src/EdFiLMS/Descriptor/AssignmentCategory.metaed [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+Descriptor AssignmentCategory
+    documentation "he category or type of assignment."
```
Comment (ggaronzik) [inline src/EdFiLMS/Descriptor/AssignmentCategory.metaed:7]: Small typo.

**Comment by ggaronzik** [comment]
@stephenfuqua, I found one small typo. Otherwise this looks good to me!

File: extension/Descriptors/AssignmentCategoryDescriptor.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (stephenfuqua) [inline extension/Descriptors/AssignmentCategoryDescriptor.xml:1]: As discussed on slack, I have not been able to get any bulk loading applications to work yet with the extension, so these XML files for descriptors are effectively untested. We'll try to resolve the issues under a new ticket.

**Review by bradbanister — APPROVED** [review]
Looks good!

**Pull request merged**